docs/content/installation/installing-nic/installation-with-manifests.md

```diff
@@ -15,7 +15,7 @@ docs: "DOCS-603"
 
 ### Get the NGINX Controller Image
 
-{{<note>}}Always use the most up-to-date stable release listed on the [releases page]({{< relref "releases.md" >}}).{{</note>}}
+{{<note>}} Always use the most up-to-date stable release listed on the [releases page]({{< relref "releases.md" >}}).{{</note>}}
```

Comment:
I don't think this space affects anything

Reply:
It doesn't do anything, but I've made a weird habit of this for scan readability.

I'll remove it now.